src/servers/graphql-server.ts

```diff
@@ -142,31 +142,39 @@ export const sessionContext = ({
   return addAttributesToCurrentSpanAndPropagate(
     {
       "token.sub": tokenPayload?.sub,
+      "token.iss": tokenPayload?.iss,
```

Comment:
I think we may not need this right now?

Reply:
not sure, appears to always be "galoy.io"

Reply:
Adding information to spans can't hurt.